source/tutorials.txt

```diff
@@ -10,12 +10,38 @@ Tutorials
    
    GitHub Issue Tracker </tutorial/backend>
    Set up JWT Authentication </tutorial/jwt>
+   Device Sync with Realm SDKs <https://www.mongodb.com/docs/realm/tutorial/>
 
-Learn how to:
+Learn how to use Atlas App Services' features and integrate external tools. 
 
-- :doc:`Write a GitHub Issue Tracker with Atlas App Services </tutorial/backend>` 
```

Comment:
For this page, you can remove the card groups and go back to this sort of "minimal" bullet list style, because we'll hide it in the Table of Contents (ToC). I would remove lines 17-45 that were added here, restore lines 16 and 17 from before, and add a line for the link to Device Sync with Realm SDKs. So we would have lines 16-18 being something like:
```
- :doc:`Write a GitHub Issue Tracker with Atlas App Services </tutorial/backend>`
- :doc:`Set up JWT Authentication with App Services </tutorial/jwt>`
- :ref:`Use Device Sync with Realm SDKs <tutorial>`
```
That last one looks different because it's using intersphinx to follow a ref target to our docs-realm repository, while the first two are links to paths within this docs-app-services repository.

Reply:
The <tutorial> link wasn't working, so I removed the ref and added the link manually.

Reply:
reopening. we should have a *bit* of content on this page in case a user finds it. 

the body can be something like what dachary proposed above: 

> - :doc:`Write a GitHub Issue Tracker with Atlas App Services </tutorial/backend>`
> - :doc:`Set up JWT Authentication with App Services </tutorial/jwt>`
> - :ref:`Use Device Sync with Realm SDKs <tutorial>`